horton/scripts/test/test_atomdb.py

```diff
@@ -196,3 +196,14 @@ def test_script_convert_g03():
         assert padb.get_numbers() == [1, 8]
         assert padb.get_charges(1) == [0, -1]
         assert padb.get_charges(8) == [+1, 0, -1]
+
+
+def test_plot_atoms():
+    padb = ProAtomDB.from_refatoms(numbers=[8, 1], max_kation=1, max_anion=1)
```

Comment:
Oops! That is one of my favorite spelling errors: a horrible Dutchism embedded in my muscle memory. Are you OK with fixing this throughout the code? I would do that in a separate PR.